internal/resource-request-operator/resourceRequest_controller.go

```diff
@@ -0,0 +1,162 @@
+package resourceRequestOperator
+
+import (
+	"context"
+	"github.com/go-logr/logr"
```

Comment:
Logr should be dropped if using klog

@palexster To avoid problems, this package could also be blacklisted with golangci-lint (as done in CrownLabs)